zeq/hbp/detail/frame.fbs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Stefan Eilemann
+//
+namespace zeq.hbp;
+
+table Frame
+{
+    /// starting frame number for an animation loop
+  start:uint;
+    /// current frame (wrapped into [start end])
+  current:uint;
+    /// end frame number for an animation loop
+  end:uint;
```

Comment:
add delta to have different increment than 1 (could even be decrement, like -2 to animate backwards 2 times the 'normal' speed)?